### DISCUSSION TITLE
I think latest update broke the cron scheduler?

### DISCUSSION THREAD

**Discussion by pierroo** [supabase/supabase#20382] [Questions] [answered]
I just did the latest upgrade from my project settings, and now I am unable to schedule a job like before.
Please see below:
![image](https://github.com/supabase/supabase/assets/17619412/87cad935-8750-4743-8f23-dc3ff04963ba)

Apparently there may be an issue with the order of parameters in the query?

But that's the order mentioned in the docs:
![image](https://github.com/supabase/supabase/assets/17619412/9c7b15f4-2f71-41ac-9242-3064000c7a30)

Not sure what's wrong?

**Comment by awalias** [comment, marked as answer]
I couldn't reproduce on my end unfortunately. Judging by the error this is most likely a permissions error with the current user and the pg_cron schema - if the problem persists you could try disabling the cron extension and re-enabling from the dashboard 🤔

**Reply by pierroo** [reply]
I thought the second line of the error was mainly about constraints on the order of parameters?
where can I disable the cron extension?

**Reply by pierroo** [reply]
Thank you so much @awalias , surprisingly disabling the cron extension and re-enabling it on public schema did the trick
although not sure if it is wise to have it on public schema? not yet familiar with what it may mean and if it could bring security issues

**Comment by Judimax** [comment]
@awalias have you been able to setup pg_cron on your local dev environment I need assistance in setting up mine heres a link to the issue


https://github.com/citusdata/pg_cron/issues/306

**Comment by dels07** [comment]
I got issue when try to disable `pg_cron`
![Screenshot 2024-02-06 at 10 22 12](https://github.com/supabase/supabase/assets/4617477/880dc6ef-f1e3-4d83-8930-487331fa0a14)

unschedule cron not working either
![Screenshot 2024-02-06 at 10 03 16](https://github.com/supabase/supabase/assets/4617477/fe36d1f3-cf8c-43d6-96eb-01b06e873997)
